controllers/common/types.go

```diff
@@ -29,6 +29,13 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	// AnnotationCleanFinalizer key
+	AnnotationCleanFinalizer = `chaos-mesh/cleanFinalizer`
```

Comment:
It seems we should define the annotation key prefix for chaos-mesh. What about `chaosmesh.pingcap.com/` @zhouqiang-cl @cwen0